config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
+  namespace :admin do
+    authenticate(:admin_user) do
```

Comment:
maybe we should we add a test to prevent regressions here?

Reply:
@santib I agree, any suggestions on where a test like this should live?

Reply:
I think any of a system/feature/request spec would be fine

Reply:
@santib I just realized that rails_api_base doesn't have Capybara to run a system spec. Is Capybara something we would like to add?

Reply:
hmm let's not add it here in this PR, we can discuss separately if we want to have system specs for our admin or not. Let's go with a request spec for now